src/Bedrock.Framework.Experimental/Client/ClientExtensions.cs

```diff
@@ -0,0 +1,13 @@
+        
+namespace Bedrock.Framework
+{
+    public static class ClientExtensions
+    {
+        public static ClientBuilder UseConnectionPooling(this ClientBuilder builder)
```

Comment:
How does one specify the pooling policy? What sort of configurability does the pool have?

Reply:
I'm not certain how someone will be able to define the pooling policy yet, but as you mentioned before, probably something similar to what ObjectPool does in the runtime.